docs/data/base/components/input/UnstyledInputIntroduction/tailwind/index.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Input } from '@mui/base/Input';
+import { useTheme } from '@mui/system';
+
+function useIsDarkMode() {
+  const theme = useTheme();
+  return theme.palette.mode === 'dark';
+}
+
+export default function UnstyledInputIntroduction() {
+  // Replace this with your app logic for determining dark modes
+  const isDarkMode = useIsDarkMode();
+
+  return (
+    <Input
```

Comment:
Let's extract this to a custom component, similar to the [`CustomTablePagination`](https://deploy-preview-38367--material-ui.netlify.app/base-ui/react-select/#tailwind-UnstyledSelectIntroduction.js) component. This applies for all PRs :)